### PR TITLE
Project Guess who - Isabella Markskog

### PR DESCRIPTION
**Pull request by IsabellaM5**
**The project**

I did all the requirements for blue and red level. 

This week was the second week of delving into the JavaScript world. I think the project turned out great JavaScript functionality wise, even though I would've liked to add a bit more of my own styling to the project.

Link to the live page: https://amazing-mccarthy-a42098.netlify.app/

**Things I struggled with**

I think I got the general idea of the game and the different steps quite well (huge thanks to the great comments in the starter code!). But writing the different functions in code, especially understanding the functions itself in the written code, was hard work and took a lot of time! I had to reiterate the written code by talking to myself what the code actually do. 

Also, I aimed at writing as less code as possible as a starter point, which onlu added to the challenge for me. What I did was to write the different functions from the basics in code to then proceeded to look up how to write code that do the same thing but with less lines of code.

### CODE REVIEW

**Review by annalenksjo — COMMENTED** [review]
My first code review! A lot to learn for me, and I will for sure get better at this. I hope you will be satisfied with my review and please reach out if something is unclear. 

I really liked your design, and your code is clean and structured. I especially liked that you really used the comments to explain the code, which made it much easier when reviewing! I will try to implement this more in my future projects :-)

**Review by annalenksjo — APPROVED** [review]

File: code/script.js [hunk 1]
```diff
@@ -10,222 +15,296 @@ const CHARACTERS = [
     img: 'images/jabala.svg',
     hairColor: 'hidden',
     eyeColor: 'hidden',
+    clothesColor: 'green',
     glasses: true,
     hat: true,
+    jewelry: false,
+    pet: false,
```
Comment (KarolCislowski) [inline code/script.js:22]: Nicely altered objects 🥇

File: code/script.js [hunk 2]
```diff
@@ -241,103 +320,128 @@ const generateBoard = () => {
...
+// This function is to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
-}
+  winOrLose.style.display = 'none';            
+  board.style.display = 'flex';
+  countUserFiltering = 0;            
+  filterCounter.innerHTML = `NUMBER OF QUESTIONS ASKED: ${countUserFiltering}`;
```
Comment (KarolCislowski) [inline code/script.js:336]: Niece and clean way to display the counter 🤓

**Review by KarolCislowski — APPROVED** [review]
This is a very impressive project! 🌈  You have shown great ambition 🦄 

Your code was very DRY and clean!

GREAT JOB!! 🥳

**Pull request closed without merging**